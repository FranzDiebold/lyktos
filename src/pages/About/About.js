import React from 'react';
import { Helmet } from 'react-helmet';

import {
    Hero, HeroBody, Container,
    Title, Subtitle,
    Columns, Column,
    Image, Content,
} from 'bloomer';

import './About.scss';

function About() {
    return (
        <div>
            <Helmet>
                { /* eslint-disable-next-line */ }
                <title>about üî± lyktos</title>
            </Helmet>

            <Image src="img/gods/gods.png" />

            <Hero className="first-section-after-title-image" isSize="medium">
                <HeroBody>
                    <Container className="has-text-centered">
                        <Title isSize="1">
                            <span role="img" aria-label="thanks">üî±</span> lyktos
                        </Title>
                        <Subtitle isSize="3">
                            Compare greek and roman gods and goddesses.
                        </Subtitle>
                    </Container>
                </HeroBody>
            </Hero>
            <Hero className="has-colored-bg" isSize="medium">
                <HeroBody>
                    <Container className="has-text-centered">
                        <Title>Project background</Title>
                        <Columns isVCentered>
                            <Column isSize={4}>
                                <Columns isCentered isVCentered isMobile>
                                    <Column isSize="narrow">
                                        <Image isSize="96x96" src="/img/about/react-logo.svg" />
                                    </Column>
                                    <Column isSize="narrow">
                                        <Subtitle isSize="1">+</Subtitle>
                                    </Column>
                                    <Column isSize="narrow">
                                        <Image isSize="96x96" src="/img/gods/greek/poseidon.svg" />
                                    </Column>
                                </Columns>
                            </Column>
                            <Column isSize={8} className="has-text-justified">
                                <Content>
                                    <p>
                                        What should you do if you both want to learn <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">React</a> and
                                        get to know <a href="https://en.wikipedia.org/wiki/Greek_mythology" target="_blank" rel="noopener noreferrer">greek</a> and <a href="https://en.wikipedia.org/wiki/Roman_mythology" target="_blank" rel="noopener noreferrer">roman</a> mythology? <br />
                                        You create <span role="img" aria-label="thanks">üî±</span> lyktos!
                                    </p>
                                    <p>
                                        It is open source, check it out on <a href="https://github.com/FranzDiebold/lyktos" target="_blank" rel="noopener noreferrer">GitHub</a>!
                                    </p>
                                </Content>
                            </Column>
                        </Columns>
                    </Container>
                </HeroBody>
            </Hero>
            <Hero className="has-white-bg" isSize="medium">
                <HeroBody>
                    <Container className="has-text-centered">
                        <Title>Etymology</Title>
                        <Columns isVCentered>
                            <Column isSize={4}>
                                <Columns isCentered>
                                    <Column isSize="narrow">
                                        <Image isSize="128x128" src={`/img/gods/greek/zeus.svg`} />
                                    </Column>
                                </Columns>
                            </Column>
                            <Column isSize={8}>
                                <Content>
                                    <p className="has-text-justified">
                                        <b>Lyktos</b>, Lyctus or Lyttos (Greek: ŒõœçŒ∫œÑŒøœÇ or ŒõœçœÑœÑŒøœÇ), was one of the most considerable cities in ancient Crete, which appears in the Homeric catalogue. Lyttos is now a village in the municipality of Minoa Pediada.
                                    </p>
                                    <p className="has-text-justified">
                                        According to Hesiod, Theogony (477-484), <b>Rhea gave birth to Zeus</b> in Lyctus and hid him in a cave of Mount Aegaeon. The inhabitants of this ancient Doric city called themselves colonists of Sparta, and the worship of Apollo appears to have prevailed there.
                                    </p>
                                </Content>
                            </Column>
                        </Columns>
                    </Container>
                </HeroBody>
            </Hero>
            <Hero isSize="medium">
                <HeroBody>
                    <Container className="has-text-centered">
                        <Title><span role="img" aria-label="thanks">üôè</span> Thank you!</Title>
                        <Content>
                            <p>
                                All information is taken from <a href="https://www.wikipedia.org" target="_blank" rel="noopener noreferrer">Wikipedia</a>.
                            </p>
                            <p>
                                Gods vector illustrations by <a href="https://www.vecteezy.com/" target="_blank" rel="noopener noreferrer">www.Vecteezy.com</a>.
                            </p>
                            <p>
                                Heaven background designed by <a href="http://www.freepik.com" target="_blank" rel="noopener noreferrer">macrovector / Freepik</a>.
                            </p>
                            <p>
                                <a href="https://bulma.io" target="_blank" rel="noopener noreferrer">
                                    <img src="https://bulma.io/images/made-with-bulma.png" alt="Made with Bulma" width="128" height="24" />
                                </a>
                            </p>
                        </Content>
                    </Container>
                </HeroBody>
            </Hero>
        </div>
    );
}

export default About;
