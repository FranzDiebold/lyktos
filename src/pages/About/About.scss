.first-section-after-title-image {
    padding-top: 0;
}
