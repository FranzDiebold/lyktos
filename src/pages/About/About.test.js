import React from 'react';
import renderer from 'react-test-renderer';

import About from './About';

it('renders about page correctly', () => {
    const tree = renderer
        .create(<About />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
