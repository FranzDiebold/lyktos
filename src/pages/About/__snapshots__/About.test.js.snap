// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about page correctly 1`] = `
<div>
  <figure
    className="image"
  >
    <img
      src="img/gods/gods.png"
    />
  </figure>
  <section
    className="hero is-medium first-section-after-title-image"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title is-1"
        >
          <span
            aria-label="thanks"
            role="img"
          >
            üî±
          </span>
           lyktos
        </h1>
        <h2
          className="subtitle is-3"
        >
          Compare greek and roman gods and goddesses.
        </h2>
      </div>
    </div>
  </section>
  <section
    className="hero is-medium has-colored-bg"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title"
        >
          Project background
        </h1>
        <div
          className="columns is-vcentered"
        >
          <div
            className="column is-4"
          >
            <div
              className="columns is-centered is-mobile is-vcentered"
            >
              <div
                className="column is-narrow"
              >
                <figure
                  className="image is-96x96"
                >
                  <img
                    src="/img/about/react-logo.svg"
                  />
                </figure>
              </div>
              <div
                className="column is-narrow"
              >
                <h2
                  className="subtitle is-1"
                >
                  +
                </h2>
              </div>
              <div
                className="column is-narrow"
              >
                <figure
                  className="image is-96x96"
                >
                  <img
                    src="/img/gods/greek/poseidon.svg"
                  />
                </figure>
              </div>
            </div>
          </div>
          <div
            className="column is-8 has-text-justified"
          >
            <div
              className="content"
            >
              <p>
                What should you do if you both want to learn 
                <a
                  href="https://reactjs.org"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  React
                </a>
                 and get to know 
                <a
                  href="https://en.wikipedia.org/wiki/Greek_mythology"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  greek
                </a>
                 and 
                <a
                  href="https://en.wikipedia.org/wiki/Roman_mythology"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  roman
                </a>
                 mythology? 
                <br />
                You create 
                <span
                  aria-label="thanks"
                  role="img"
                >
                  üî±
                </span>
                 lyktos!
              </p>
              <p>
                It is open source, check it out on 
                <a
                  href="https://github.com/FranzDiebold/lyktos"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  GitHub
                </a>
                !
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="hero is-medium has-white-bg"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title"
        >
          Etymology
        </h1>
        <div
          className="columns is-vcentered"
        >
          <div
            className="column is-4"
          >
            <div
              className="columns is-centered"
            >
              <div
                className="column is-narrow"
              >
                <figure
                  className="image is-128x128"
                >
                  <img
                    src="/img/gods/greek/zeus.svg"
                  />
                </figure>
              </div>
            </div>
          </div>
          <div
            className="column is-8"
          >
            <div
              className="content"
            >
              <p
                className="has-text-justified"
              >
                <b>
                  Lyktos
                </b>
                , Lyctus or Lyttos (Greek: ŒõœçŒ∫œÑŒøœÇ or ŒõœçœÑœÑŒøœÇ), was one of the most considerable cities in ancient Crete, which appears in the Homeric catalogue. Lyttos is now a village in the municipality of Minoa Pediada.
              </p>
              <p
                className="has-text-justified"
              >
                According to Hesiod, Theogony (477-484), 
                <b>
                  Rhea gave birth to Zeus
                </b>
                 in Lyctus and hid him in a cave of Mount Aegaeon. The inhabitants of this ancient Doric city called themselves colonists of Sparta, and the worship of Apollo appears to have prevailed there.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="hero is-medium"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title"
        >
          <span
            aria-label="thanks"
            role="img"
          >
            üôè
          </span>
           Thank you!
        </h1>
        <div
          className="content"
        >
          <p>
            All information is taken from 
            <a
              href="https://www.wikipedia.org"
              rel="noopener noreferrer"
              target="_blank"
            >
              Wikipedia
            </a>
            .
          </p>
          <p>
            Gods vector illustrations by 
            <a
              href="https://www.vecteezy.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              www.Vecteezy.com
            </a>
            .
          </p>
          <p>
            Heaven background designed by 
            <a
              href="http://www.freepik.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              macrovector / Freepik
            </a>
            .
          </p>
          <p>
            <a
              href="https://bulma.io"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Made with Bulma"
                height="24"
                src="https://bulma.io/images/made-with-bulma.png"
                width="128"
              />
            </a>
          </p>
        </div>
      </div>
    </div>
  </section>
  <section
    className="hero is-medium has-white-bg"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title"
        >
          <span
            aria-label="speech bubble"
            role="img"
          >
            üí¨
          </span>
           
          <i>
            "Alexa, open 
            <span
              className="has-text-primary"
            >
              mythology land
            </span>
            "
          </i>
        </h1>
        <div
          className="columns is-vcentered"
        >
          <div
            className="column is-4"
          >
            <div
              className="columns is-centered"
            >
              <div
                className="column is-narrow"
              >
                <figure
                  className="image is-128x128"
                >
                  <img
                    src="/img/mythology-land-alexa-skill-preview-image.png"
                  />
                </figure>
              </div>
            </div>
          </div>
          <div
            className="column is-8"
          >
            <div
              className="content"
            >
              <p
                className="has-text-justified"
              >
                Also check out the corresponding 
                <b>
                  Alexa skill
                </b>
                 "mythology land", which is also 
                <a
                  href="https://github.com/FranzDiebold/mythology-land-alexa-skill"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  open source
                </a>
                !
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
